authors/apps/authentication/tests/base_test.py

```diff
@@ -29,4 +28,10 @@ def __init__(self):
                                           "password": self.password,
                                           }
                                  }
+
+        self.no_password_login = {
+            "user": {"email": self.user_email,
+                     "password": None,
```

Comment:
Do we really need to pass non here?

Reply:
I was still figuring out the way to get the validation error,